src/components/shared/AsyncButton.svelte

```diff
@@ -20,7 +20,7 @@
     export let classes = "";
 </script>
 
-<button on:click={(e) => { promise = handleClick(e)}} class="{classes} {active ? 'active' : ''}">
+<button on:click={(e) => { promise = handleClick(e)}} class="{classes} {active ? 'bg-slate-500' : 'bg-slate-400'} w-14">
```

Comment:
another example of logic changing for the worse with pure tailwind. before, this button could be used anywhere and what "active" means could easily be context specific. now it's stateful ui (colors) is hardwired unless i add more tangled logic here.